File: backend/PaymentService/app/domain/entities/order.js
class User {
	#product_id
	#amount
	#order_id

	constructor(order_id, product_id, amount) {
		this.#product_id = product_id;
		this.#order_id = order_id;
		this.#amount = amount;
	}

	getId() {
		return this.#order_id;
	}

	getProductId() {
		return this.#product_id;
	}

	getAmount() {
		return this.#amount;
	}
}

File: backend/OrderManagementService/app/domain/entities/order.py
class Order:
    def __init__(self, order_id, product_id, amount):
        self._order_id = order_id
        self._product_id = product_id
        self._amount = amount

    def getId(self):
        return self._order_id

    def getProductId(self):
        return self._product_id

    def getAmount(self):
        return self._amount
    
    def __str__(self) -> str:
        return str({
            'order_id': self.getId(),
            'product_id': self.getProductId(),
            'amount': self.getAmount()
        })